src/components/BaseKeymap.vue

```diff
@@ -45,6 +44,13 @@ export default {
         max.y *= this.config.SCALE;
       }
       return max;
+    },
+    calculateMaxFromLayout(layout) {
+      return this._calculateMax(layout);
+    },
+    calculateMax(newLayout) {
```

Comment:
not sure about this. just leave it in the tester keymap version. It has no value in the base keymap